### ISSUE TITLE
[Feature Request]: option to highlight items with ETV over $100

### ISSUE THREAD

**Issue by LitBomb** [FMaz008/VineHelper#164]
### Feature Description

I was wondering if it is possible to have an option to highlight items with ETV over $100, similar to the feature to highlight items that are recently added using the "Now", "-3Hrs", "-12Hrs", and "-24Hrs" buttons.  Thanks!
![image](https://github.com/user-attachments/assets/87b08071-3024-475a-96b2-c2d42ce6393c)

**Comment by stacyqualls** [comment]
I like this idea as well, but It might be simpler to create tabs for Silver and Gold tiers.

**Comment by FMaz008** [comment]
Not exactly what you've suggested, but the new keyword system will allow you to define an etv_min and etv_max value for your keywords, of you want to. 

24d24619f4617afa9670167f09f784c4fb7bbfb0

**Comment by LitBomb** [comment]
Hi, right now I have a script I insert into the browser console to highlight the item price if the item is over $100 (for Vine Gold users).  Here is a screenshot:

![image](https://github.com/user-attachments/assets/85ad51b6-a455-4340-919c-173ae2277aac)

I want to be able to quickly see items that are over $100 among all items on the notification monitor page.

**Comment by LitBomb** [comment]
Another additional feature would be to make a sound when an item over $100 is available on the VH monitor so I could quickly check the VH monitor page to look for the highlighted item.

**Comment by FMaz008** [comment]
Do you use the highlight keyword at all ? Because if you don't you will be able to dedicate them for that purpose and they already allow to set a sound for them.

What you can't do is set a different sound for every keyword, of highlight in a different way for ETV compared to the regular highlighted items.

**Comment by LitBomb** [comment]
Yes, I do use keywords to highlight items I am looking for.  I want to be able to also highlight any items that is over $100 whether they match my keywords or not.

**Comment by LitBomb** [comment]
Thanks for the pointers.  I think I didn't explain it well previously.

What I would like is the ability to get notify when an item with a ETV $100 or higher becomes available.  There is no keyword matching involved.  For example:

☑ Highlight any item >$100 ETV

I imagine a lot of gold viners would like to know when a more expensive item that only gold viners can order becomes available, regardless of whether it matches a keyword or not.  For example, with my injected javascript, I ordered a 6000A jump starter ($169) and an outdoor long range TV antenna ($149)  These items I didn't know I wanted, but my script alerted me when they showed up and I ordered them because I realized I could use them.

The javascript that I inject into the browser dev console now does these:
- when the page is updated:
- find values larger than 100 that is previously not alerted
- highlight the number in red, make its font larger
- make a beep (just a beep, different than other VH sounds)
- remember previously alerted >$100 items
- repeat

I understand if this is too niche.  I will continue to re-insert my javascript into the browser when I refresh the notification monitor page.  I do love VH very much!

Thanks.

**Comment by FMaz008** [comment]
If you have the skills to run your own JS code, you can probably run the github code , which allow you to make your own plugin and it would load by items and could be listening to the newItem event.

I understand your suggestion, what I'm saying is that the new keyword system get you halfway there by creating a keyword that match everything AND an ETV_MIN value of 100.

It just doesn't differenciate the highlight from other keywords that may match.
